simulation_parameters/microbe_stage/biomes.json

```diff
@@ -1511,7 +1535,11 @@
         "z": 0.75
       }
     },
+    "WaterCurrentSpeed": 1.5,
+    "WaterCurrentChaoticness": 2.0,
+    "WaterCurrentScale": 1.0,
     "CompoundCloudBrightness": 2.5,
```

Comment:
This block is inconsistent with the other ones where all of the new properties are next to each other without gaps.